### PR TITLE
Fix/issue 31 fullpage scroll

### PR DESCRIPTION
**Pull request by wswebcreation**
This release fixes 

- https://github.com/wswebcreation/webdriver-image-comparison/issues/31
- https://github.com/wswebcreation/webdriver-image-comparison/issues/30

### CODE REVIEW

File: lib/clientSideScripts/scrollToPosition.spec.ts [hunk 1]
```diff
@@ -1,27 +0,0 @@
-import scrollToPosition from './scrollToPosition';
-
-const CONFIGURABLE = {
-  writable: true,
-  configurable: true,
-};
-
-describe('scrollToPosition', () => {
```
Comment (anto-ac) [inline lib/clientSideScripts/scrollToPosition.spec.ts]: Wouldn't you want to test the new implementation even if it's much simpler?

File: lib/clientSideScripts/hideScrollbars.ts [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
-  } else {
-    document.body.style.overflow = '';
+  try {
+    if (hide) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
+    }
+  } catch (e) {
+    // Do nothing
```
Comment (anto-ac) [inline lib/clientSideScripts/hideScrollbars.ts]: Is it worth logging something here?

Comment (wswebcreation) [reply lib/clientSideScripts/hideScrollbars.ts]: This is logged into browser where the test is executed, it doesn't return anything

File: lib/clientSideScripts/scrollToPosition.spec.ts (hunk 1 above)
Comment (wswebcreation) [reply lib/clientSideScripts/scrollToPosition.spec.ts]: This is default browser implementation and imho not worth testing, it's not risk in my opinion

Comment (anto-ac) [reply lib/clientSideScripts/scrollToPosition.spec.ts]: Fair enough, but there is a tiny bit of logic :-)

Comment (wswebcreation) [reply lib/clientSideScripts/scrollToPosition.spec.ts]: Haha, yeah, there is, but not a big risk

**Comment by wswebcreation** [comment]
I've added documentation to explain the issue I'm solving with they `try/catch`, if you agree can you approve it?

**Pull request merged**